Packs/Hoxhunt/Integrations/Hoxhunt/Hoxhunt.yml

```diff
@@ -0,0 +1,179 @@
+category: Messaging
+commonfields:
+  id: Hoxhunt
+  version: -1
+configuration:
+- defaultvalue: https://api.hoxhunt.com/graphql-external
+  display: Server URL (e.g. https://api.hoxhunt.com/graphql-external)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: Input your api key from Hoxhunt
+  display: API Key
+  name: api_key
+  required: false
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: ''
+  name: fetch_time
+  required: false
+  type: 0
+- additionalinfo: Input a query filter for incidents to be fetched
+  display: ''
+  name: queryfilter
+  required: false
+  type: 0
+display: Hoxhunt
+name: Hoxhunt
+script:
+  commands:
+  - arguments: []
+    description: Gets the current user information from Hoxhunt.
+    name: hoxhunt-get-current-user
+    outputs:
+    - contextPath: HoxHunt.CurrentUser
+      description: Current User information from Hoxhunt.
+      type: string
+  - arguments:
+    - description: Input here the filter or other arguments.
+      name: filter
+      type: textArea
+    description: Gets incidents from Hoxhunt.
+    name: hoxhunt-get-incidents
+    outputs:
+    - contextPath: Hoxhunt.Incidents
+      description: Incidents from Hoxhunt.
+      type: string
+  - arguments: []
```

Comment:
Please add `filter` to the commands arguments, you need it in the py code (line 256)